bin/network-monitor/src/config.rs

```diff
@@ -109,6 +105,16 @@ pub struct MonitorConfig {
     )]
     pub wallet_filepath: PathBuf,
 
+    /// Enable the counter increment task (optional; disabled by default).
+    #[arg(
+        long = "enable-counter",
+        env = "MIDEN_MONITOR_ENABLE_COUNTER",
+        action = clap::ArgAction::SetTrue,
+        default_value_t = false,
+        help = "Whether to enable the counter increment task"
```

Comment:
nit: Here too I think it would be nice to specify that the counter contract is a network account that increases a counter through network notes